app/scripts/components/granules/overview.js

```diff
@@ -73,7 +74,9 @@ class GranulesOverview extends React.Component {
       }
     };
     const { granules } = this.props;
-    return bulkActions(granules, config);
+    return bulkActions(granules, config).concat(
```

Comment:
This may not be the best way to compose the two functions - I'm open to suggestions.

This way _is_ expandable, though. Any new additions (buttons) can be added by adding them to the `concat` statement after creating an analogous `recoverAction`-esque function.

Reply:
To me, I think I prefer that the logic controlling what will be rendered (via what gets added to the returned array) here is in this file. For example

```
  generateBulkActions () {
    const actionConfig = {
      execute: {
        options: this.getExecuteOptions(),
        action: this.applyWorkflow
      }
    };
    const { granules, config } = this.props;
    let actions = bulkActions(granules, actionConfig);
    if (config.recoveryPath) {
      actions.concat(recoverAction(granules, actionConfig));
    }
    return actions;
  }
```

Reply:
LGTM :+1:

Reply:
https://github.com/nasa/cumulus-dashboard/pull/568/commits/e2407e9d5aeb25d257b39f14f4ca266d34bc21ef